doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+
+   .. note::
+
+      If you activated your iBases SIM card before, click :guilabel:`Skip this step`.
+
+      If you are using a SIM card from another provider, make sure you activate it through your network operator, then click :guilabel:`Skip this step`.
+
+   After the SIM card is activated, you are redirected to the :guilabel:`Add LTE Device` screen.
+   Leave the browser window open and continue with the next step before you enter the information on this screen.
+
+#. Punch out the nano-SIM from the SIM card and plug it into the SIM card holder on the nRF9160 DK.
+#. **Optional:** If you want to test the optional GPS functionality and you have an external GPS antenna for your nRF9160 DK, attach it to connector **J2** to the left of the LTE antenna.
+   See `nRF9160 DK GPS`_ for more information.
+#. Connect the nRF9160 DK to the computer with a USB cable and turn it on, or reset the device if it is already turned on.
+#. Wait up to three minutes for the device to find the cellular network and connect to the nRF Cloud server.
+
+   At this stage, the device is provisioned on nRF Cloud, but not yet associated with your nRF Cloud account.
+   When the DK has connected, the **LED3** double pulse blinks to indicate that user association is required and you can move to the next step.
+
+#. In the :guilabel:`Add LTE Device` screen from Step 4, enter your device ID and ownership code.
+
+   .. figure:: /images/nrfcloud_add_lte_device.png
+      :alt: nRF Cloud - Add LTE Device window
+
+      nRF Cloud - Add LTE Device window
+
+   * **Device ID:** the device ID is composed of *nrf-* and the 15-digit :term:`International Mobile (Station) Equipment Identity (IMEI)` number that is printed on the label of your nRF9160 DK.
+     For example, *nrf-123456789012345*.
+   * **Owndership code:** the ownership code is the PIN or the hardware ID of your DK, and it is found on the label of your nRF9160 DK.
+
+     If the label contains a PIN in addition to the IMEI number, enter this pin.
+     If it does not contain a PIN, enter the Hardware ID (HWID) HEX code, with or without colons.
+     For example, *AA:BB:CC:DD:EE:FF* or *AABBCCDDEEFF*.
+
+     .. note::
+
+        The ownership code serves as a password and proves that you own the specific nRF9160 DK.
+        Therefore, you should not share it with anyone.
+
+#. Click :guilabel:`Add Device` and wait for the device to reconnect to nRF Cloud.
+   It is normal for the device to disconnect and reconnect multiple times during device provisioning.
+
+The nRF9160 DK is now added to your nRF Cloud account.
+This is indicated by the blinking of **LED3** on the DK, which shows that it is publishing data.
+See :ref:`Asset Tracker v2 LED indication <led_indication>` for more information.
+
+On nRF Cloud, you can access the device by clicking :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left.
+
+.. _nrf9160_gs_testing_dk:
+
+Testing the DK
```

Comment:
Should this be "Testing the application"?

Reply:
I see this as testing the DK, but am open to other interpretations.

Reply:
The original talks about testing the application. Testing the actual DK might be a wider concept than just these four steps.

Reply:
This section also contains the subsections for testing the cellular connection and the GNSS functionality.